TWC_Backend/TWC_DatabaseLayer/DTOs/ProjectMemberDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace TWC_DatabaseLayer.DTOs
+{
+    public class ProjectMemberDTO
+    {
+        [Required]
+        public int ProjectId { get; set; }
+        [Required]
+        public int UserId {  get; set; }
+    }
+}
```

Comment:
This class is not needed, since the id and bool isOwner can be ignored

Reply:
IsOwner is ignored to secure it from possible http request mistakes. IsOwner is true only when User creates Project, and any other Users can be only members, so I think it's unnecessary to send field "false" every time